hooks/models.js

```diff
@@ -44,7 +44,7 @@ module.exports = function Models(cb) {
         }
 
         // hand off to adapter
-        adapter.registerModel(connection.connection, modelDefinition, function(err, model) {
+        adapter.registerModel(connection.connection, model.definition, function(err, model) {
```

Comment:
the `modelDefinition` in this case is the entire `module.exports` from each model file. This allows each adapter to specify the appropriate top level keys for its own purpose. For example, the mongoose adapter uses a `schema` attribute which holds the actual model definition, and then has other top level attributes like `schemaOptions`, `methods`, `statics`, `virtuals`, `middleware`, which are used for other purposes like defining instance/class methods, virtual properties, and lifecycle hooks on each model. I think in the case with `vogels` adapter, the adapter would check for a `definition` attribute on the modelDefinition